google-datacatalog-rdbms-connector/tests/google/datacatalog_connectors/rdbms/prepare/sql_objects/sql_objects_datacatalog_tag_factory_test.py

```diff
@@ -38,7 +36,7 @@ def test_tag_for_function_sql_object_should_set_all_available_fields(  # noqa:E5
         sql_objects_config = \
             utils.Utils.convert_json_to_object(self.__MODULE_PATH,
                                                'sql_objects_config.json')
-        factory = sql_objects_datacatalog_tag_factory.\
+        factory = sql_objects.sql_objects_datacatalog_tag_factory.\
```

Comment:
```suggestion
        factory = sql_objects.SQLObjectsDataCatalogTagFactory(sql_objects_config)
```
Please replace all similar occurrences to turn the code less verbose if it makes sense.